docs/angular/add-to-existing.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Add to Existing Angular Project
+sidebar_label: Add to Existing
+---
+
+import DocsCard from '@components/global/DocsCard';
+import DocsCards from '@components/global/DocsCards';
+
+<head>
+  <title>Add Ionic Angular to Existing Project: Integration Guide</title>
+  <meta
+    name="description"
+    content="Learn how to add Ionic Angular to your existing Angular project. Step-by-step guide for integrating Ionic components and features into an existing Angular application."
+  />
+</head>
+
+This guide covers how to add Ionic Angular to an existing Angular project. If you're looking to start a new project from scratch, check out the [Ionic Angular Quickstart](/docs/angular/quickstart.md) guide. For an overview of how Ionic Angular works with Angular, including version support and tooling, check out the [Ionic Angular Overview](/docs/angular/overview.md).
+
+:::tip
+
+This guide uses `.css` file extensions for stylesheets. If you created your Angular app with a different stylesheet format (such as `.scss`, `.sass`, or `.less`), use that extension instead.
+
+:::
+
+## Setup
+
+You can add Ionic Angular to your existing Angular project using the Angular CLI's `ng add` feature or by installing it manually.
+
+### Using ng add
+
+The easiest way to add Ionic Angular is to use the Angular CLI's `ng add` feature:
+
+```bash
+ng add @ionic/angular
+```
+
+This will install the `@ionic/angular` package and automatically configure the necessary imports and styles.
+
+### Manual Installation
+
+If you prefer to install Ionic Angular manually, you can follow these steps:
+
+#### 1. Install the Package
+
+```bash
+npm install @ionic/angular
+```
+
+#### 2. Add Ionic Framework Stylesheets
+
+Replace the existing `styles` array in `angular.json` with the following:
+
+```json title="angular.json"
```

Comment:
Woah, didn't know we could add a title and it would display! It would have been great for the Your First App pages. I'll keep this in mind!

Reply:
Yeah I just discovered it when editing the "Add to Existing" for React. 😂 I think we could improve its styles and apply it everywhere that references updating a file.

Reply:
I agree. It should be pretty easy to update the styles for it to match.

Reply:
@thetaPC Added a PR to match Docusaurus here: https://github.com/ionic-team/ionic-docs/pull/4323